src/Symfony/Component/HttpFoundation/Session/Storage/Handler/PdoSessionHandler.php

```diff
@@ -178,8 +178,15 @@ public function __construct(#[\SensitiveParameter] \PDO|string $pdoOrDsn = null,
         $this->ttl = $options['ttl'] ?? null;
     }
 
-    public function configureSchema(Schema $schema, \Closure $isSameDatabase): void
+    /**
+     * Adds the Table to the Schema if it doesn't exist.
+     *
+     * @param \Closure $isSameDatabase
+     */
+    public function configureSchema(Schema $schema/*, \Closure $isSameDatabase*/): void
```

Comment:
the method is new to 6.3, no need for BC layers, we can declare the argument normally